tools/validate_configs/test_configs/exascaler-existing-vpc.yaml

```diff
@@ -27,9 +27,3 @@ deployment_groups:
   modules:
   - id: network1
     source: modules/network/pre-existing-vpc
```

Comment:
This blueprint is named: Exascaler existing vpc, as we remove exascaler from it, should we just remove it? 
IF we remove it, then let's make sure there is a "managed-lustre-existing-vpc" tests, otherwise, add managed lustre here and rename...